guides/k8-akka-cluster/src/main/paradox/index.md

```diff
@@ -0,0 +1,570 @@
+# Deploying Akka application on Kubernetes
+
+## The problem
+
+You've created an application using [Akka](http://akka.io/) with [Akka Clustering](http://doc.akka.io/docs/akka/current/scala/common/cluster.html) enabled. After evaluating all of your deployment options, you've chosen to deploy to [Kubernetes](https://kubernetes.io/) to leverage the facilities it provides for automated deployment, scaling, and management of containerized applications. This raises a problem though: how do you deploy an Akka application on Kubernetes? If Akka cluster is required by the application, this requires one or more seed node to be specified to bootstrap the cluster. How do we setup Akka cluster when deploying to Kubernetes? Finally, running an application on Kubernetes requires containerization. How exactly do we deploy these systems to Kubernetes?
+
+## The solution
+
+This guide will cover the steps required to deploy an Akka based application to Kubernetes.
+
+_If your application is based on Lagom or Play, refer to [Deploying Microservices to Kubernetes](http://todo-linkfor information on deploying it to Kubernetes, including how to configure the Cassandra persistence and service discovery between Lagom apps. It is worth noting that the Akka cluster setup for Lagom based applications follows the same steps outlined by this guide._
+
+As part of Kubernetes deployment, an application must be containerized. As such, this guide will show how to containerize the application using [SBT](http://www.scala-sbt.org/) or [Maven](https://maven.apache.org/).
+
+This guide will also show how to configure Kubernetes resources, particularly [StatefulSet](https://kubernetes.io/docs/concepts/workloads/controllers/statefulset/) to establish Akka cluster for your application.
+
+### Prerequisites
+
+* JDK8+
+* [Docker](https://www.docker.com/)
+* Either [SBT](http://www.scala-sbt.org/) or [Maven](https://maven.apache.org/)
+* An existing, running Kubernetes cluster.
+* The kubernetes CLI tool `kubectl` is installed and configured to point to the existing Kubernetes cluster.
+* Docker environment variables are configured to point to Docker registry used by Kubernetes cluster. This will ensure the Docker images we built in this guide will be available to the Kubernetes cluster.
+* An existing Akka based application to deploy, built using SBT or Maven.
+
+### Our approach
+
```

Comment:
I'm not sure whether I can put the solution in a diagram since I'm struggling to explain how Akka cluster works in this context using a diagram:

- Node 0 joins itself.
- Node 1 joins Node 0.
- Node 2 joins Node 0.